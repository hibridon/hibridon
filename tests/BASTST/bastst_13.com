inp=bastst_13.inp
out=bastst_13.out
run
save=save_13.inp