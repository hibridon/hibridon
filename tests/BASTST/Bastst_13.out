      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** HOMO+3P system parameters:
 NTERM  =   1        IOP    =   0        JMAX   =   2
 BROT   =  59.33     ASO1   = -68.71     ASO2   = -227.0    
 LAMMIN:    1
 LAMMAX:    9
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT:  4; JOUT:  0   1   2   3
 INDOUT:    0
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 05:13
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: c7a8ce9e36ebc7e7608468e680c6398d849b6cf3
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 05:13:49 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS= 1

 **  CC MOL + 3P ATOM ** RMU=  16.4700       E=  5000.00   JTOT=   20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   JA(IS) JMOL(J)   K   CENT    EINT(CM-1)
   1      2     0       2   380         0.000
   2      2     0       2   462         0.000
   3      1     0       1   380       158.265
   4      1     0       1   462       158.265
   5      2     2       1   380       356.004
   6      2     2       1   462       356.004
   7      2     2       2   380       356.004
   8      2     2       2   462       356.004
   9      2     2       3   380       356.004
  10      2     2       3   462       356.004
  11      2     2       3   306       356.004
  12      2     2       3   380       356.004
  13      2     2       4   462       356.004
  14      2     2       4   552       356.004
  15      2     2       4   380       356.004
  16      2     2       4   462       356.004
  17      1     2       3   380       514.269
  18      1     2       3   462       514.269
  19      1     2       3   306       514.269
  20      1     2       3   380       514.269
  21      1     2       1   462       514.269
  22      1     2       1   552       514.269
  23      1     2       2   306       514.269
  24      1     2       2   380       514.269
  25      0     2       2   462       582.981
  26      0     2       2   552       582.981
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS    40
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    0.7 %
      CURRENT DATE:  Mar 19 05:13:49 2023
 ===============================================================================
